extensions/amp-video/0.1/amp-video.js

```diff
@@ -765,7 +765,18 @@ export class AmpVideo extends AMP.BaseElement {
    */
   setUpCaptions_() {
     const captionsId = this.element.getAttribute('captions-id');
-    if (!captionsId) {
+    // if (!captionsId) {
```

Comment:
This wouldn't work beacuse you need to lazy install the `amp-story-captions` extension

Reply:
My bad this is WIP/sketchy code meant for a different PR. 
I'm saving these comments and moving this into another PR.